.github/workflows/pull-request.yaml

```diff
@@ -26,4 +26,12 @@ jobs:
         run: sudo make validate
 
       - name: Run Hull tests
-        run: cd tests && go test -v ./...
\ No newline at end of file
+        run: cd tests && go test -v ./...
+
+      - name: Check container images 
+        run: make check-images
+        if: startsWith(github.ref, 'refs/heads/upstream/release-v')
```

Comment:
This needs to run for every PR so we catch the errors beforehand and not until the release date of charts.